engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/discovery-create.html

```diff
@@ -3,89 +3,76 @@
 <form name="productForm" class="col-sm-5" ng-disabled="creating()" novalidate role="form">
 
   <section>
+    <div bst-alerts error-messages="productErrorMessages"></div>
+
     <h4 translate>Create Repositories Within:</h4>
 
-    <div class="form-group">
+    <div bst-form-group>
       <label class="checkbox-inline">
         <input type="radio"
-               name="newProduct"
-               value="false"
-               ng-model="createRepoChoices.newProduct"
-               ng-disabled="creating()"/>
+        name="newProduct"
+        value="false"
+        ng-model="createRepoChoices.newProduct"
+        ng-disabled="creating()"/>
         <span translate>Existing Product</span>
       </label>
 
       <label class="checkbox-inline">
         <input type="radio"
-               name="newProduct"
-               value="true"
-               ng-model="createRepoChoices.newProduct"
-               ng-disabled="creating()"/>
+        name="newProduct"
+        value="true"
+        ng-model="createRepoChoices.newProduct"
+        ng-disabled="creating()"/>
         <span translate>New Product</span>
       </label>
     </div>
 
-    <div class="form-group">
-      <div class="col-md-5" ng-hide="createRepoChoices.newProduct">
+
+    <div ng-hide="createRepoChoices.newProduct === 'true'">
```

Comment:
@waldenraines The problem is that `newProduct` comes from the HTML tag value at lines 14 and 23. Those are sent as strings to angular and I don't think they can be sent as booleans. 

To auto parse them as booleans I could write a directive like http://jsfiddle.net/zk99k4fg/1/ but I think it's overkill in this case (what do we gain from checking the string vs a boolean in this case?)